### PR TITLE
Custom title when creating a new pull request

### PR DESCRIPTION
**Pull request by RossValler**
Adds a new feature to input a custom pull request title as reported here: #338

### CODE REVIEW

File: src/configuration.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export interface Configuration {
   refreshPullRequestStatus: number;
   remoteName: string;
   upstream?: string;
+  customPullRequestTitle: boolean;
```
Comment (KnisterPeter) [inline src/configuration.ts]: The `package.json` configuration for the extensions as required by vscode is missing.
Please add it.

**Comment by KnisterPeter** [comment]
@RossValler Awesome. Glad for your contribution. :smiley:

File: src/configuration.ts (hunk 1 above)
Comment (RossValler) [reply src/configuration.ts]: Oh whoops, new to the whole extension thing. I'll add that right now 👍

**Comment by RossValler** [comment]
@KnisterPeter Fixed :)

**Comment by KnisterPeter** [comment]
Great. Thank you. :rocket:

**Pull request merged**